src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7839,4 +7839,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_ImplicitIndexerInitializer" xml:space="preserve">
     <value>implicit indexer initializer</value>
   </data>
+  <data name="WRN_ConvertingLock" xml:space="preserve">
+    <value>A value of type 'System.Threading.Lock' converted to another type will use likely unintended monitor-based locking in 'lock' statement.</value>
+  </data>
+  <data name="WRN_ConvertingLock_Title" xml:space="preserve">
+    <value>A value of type 'System.Threading.Lock' converted to another type will use likely unintended monitor-based locking in 'lock' statement.</value>
+  </data>
+  <data name="ERR_BadSpecialByRefLock" xml:space="preserve">
+    <value>A lock statement scope type '{0}' cannot be used in async methods or async lambda expressions.</value>
```

Comment:
>A lock statement scope type '{0}' cannot be used in async methods or async lambda expressions. [](http://example.com/codeflow?start=11&length=94)

"A lock statement on a value of  'System.Threading.Lock' type cannot be used in async methods or async lambda expressions."?